charts/datadog/templates/NOTES.txt

```diff
@@ -125,6 +125,59 @@ Trace Agent liveness probe port ({{ $liveness.port }}) is different from the con
 The Datadog Agent is listening on port {{ $apmPort }} for APM service.
 {{- end }}
 
+{{- if and .Values.datadog.apm.instrumentation.enabled_namespaces .Values.datadog.apm.instrumentation.disabled_namespaces }}
+
+###################################################################################
+####               ERROR: APM Single Step Instrumentation misconfiguration     ####
+###################################################################################
+
+The options `datadog.apm.instrumentation.enabled_namespaces` and `datadog.apm.instrumentation.disabled_namespaces` cannot be set together.
```

Comment:
to make it more visible to the user that he made a config mistake. 
you can use the `fail` feature like here: https://github.com/DataDog/helm-charts/blob/694417d7448e91ea757977acfa8ef632b4602bde/charts/datadog/templates/NOTES.txt#L262
to make the `helm install/upgrade` failing. and the error will be more visible.